apps/web/src/components/QualifiedRecruitmentCard/QualifiedRecruitmentCard.tsx

```diff
@@ -138,16 +138,14 @@ const QualifiedRecruitmentCard = ({
               <span>
                 {isOpen
                   ? intl.formatMessage({
-                      defaultMessage:
-                        "Hide this recruitment's skill assessments",
-                      id: "+h7iQH",
+                      defaultMessage: "Hide this process's skill assessments",
+                      id: "JeAzDb",
                       description:
                         "Button text to hide a miscellaneous qualified recruitment's skill assessments",
                     })
                   : intl.formatMessage({
-                      defaultMessage:
-                        "Show this recruitment's skill assessments",
-                      id: "BFewIk",
+                      defaultMessage: "Show this process's skill assessments",
```

Comment:
If this is changed, the test instances need to be changed as well.

I don't love the use of the possessive form on a word that ends with two esses (process's) and the precedes another word that starts with ess because it is awkward to pronounce. An alternative could be **Show the skill assessments of this process**, but **Hide this process's skill assessments** would also need to be changed. The French translations of these strings are essentially this alternative. Just a suggestion.

## Screenshot
<img width="629" alt="Screen Shot 2024-03-15 at 10 00 06" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/1c4bcbde-5d21-4913-bd76-4f7ef9be9e96">

Reply:
Addressed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9759/commits/01a9ca89af5f16e7d2535a6822282cd11acecf2d